packages/client/src/client/styleguide/sections/components/AtomsChart.tsx

```diff
@@ -1,96 +1,25 @@
-import { FC } from "react"
 import styled from "styled-components"
 
 import { HistoricalGraphComponent } from "@/client/App/LiveRates/Tile/HistoricalGraph"
 
 const PATH =
```

Comment:
have we already had a huge chart PATH in another component - chance for sharing?